lib/posix-futex/tests/test_posix_futex.c

```diff
@@ -109,7 +109,7 @@ static int futex(uint32_t *uaddr, int futex_op, uint32_t val,
  * Wait for the futex value to be equal to the iteration number and then
  * increment a variable.
```

Comment:
Hi @noureddine-taleb,
Please include the description in your commit message,
This description generally includes overview of changes, need of changes and general background

Reply:
ok, I've added more description to the commit message.